galgebra/metric.py

```diff
@@ -658,12 +646,12 @@ def __init__(self, basis, **kwargs):
                         printer.oprint('X_{i}', X, 'D_{i}X_{j}', dX)
 
         else:  # metric is symbolic or list of lists of functions of coordinates
-            if isinstance(g, str):  # metric elements are symbols or constants
+            if utils.isstr(g):  # metric elements are symbols or constants
                 if g == 'g':  # general symbolic metric tensor (g_ij functions of position)
                     g_lst = []
                     g_inv_lst = []
-                    for coord1 in self.coords:
-                        i1 = str(coord2)
```

Comment:
Incorrect local variable naming, probably just a copy-paste bug. It was not covered in previous tests so it was not detected. Now it's covered and passed.

Reply:
great, its really nice the tests are running well